lambdas/db-migration/src/migrations/20201014105024_create_rules_table.ts

```diff
@@ -0,0 +1,66 @@
+import * as Knex from 'knex';
+
+export const up = async (knex: Knex): Promise<void> =>
+  knex.schema.createTable('rules', (table) => {
+    table
+      .increments('cumulusId')
+      .primary();
+    table
+      .text('name')
+      .comment('Rule name')
+      .notNullable();
+    table
+      .text('workflow')
+      .comment('Workflow name to invoke for this rule')
+      .notNullable();
+    table
+      .integer('collectionCumulusId')
```

Comment:
I'm not particularly in favor of one way or another, but on your diagram this is called `cumulusCollectionId`. We should try to keep things consistent. If you think this is the better name, then we should update the diagram.

Reply:
Yeah, I think I prefer this naming convention. I'll update the diagram